From ab063972b951f160fdbc516f3d8b2d598cf090ee Mon Sep 17 00:00:00 2001
From: Dmitry Lavnikevich <dmitry.lavnikevich@scalys.by>
Date: Tue, 28 Jul 2020 17:07:46 +0300
Subject: [PATCH 13/20] Make libOpenVG.so into symlink to suppress ldconfig
 error message

Currently, libOpenVG.so is a copy of libOpenVG.3d.so. In reality it should be
a symlink to select an appropriate libOpenVG implementation depending on GPU.
---
 tools/flex-builder | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/flex-builder b/tools/flex-builder
index aacf15f..1c7f373 100755
--- a/tools/flex-builder
+++ b/tools/flex-builder
@@ -1187,6 +1187,7 @@ merge_components() {
 	   sudo sed -i "2 i\ $APPS_REPO_LIST" $RFSDIR/etc/packages.list
 	   sudo sed -i "3 i\ " $RFSDIR/etc/packages.list
 	fi
+	ln -sf libOpenVG.3d.so $RFSDIR/usr/local/lib/libOpenVG.so
 	[ $VIRTABLE = y ] && sudo chroot $RFSDIR ldconfig
     elif [ $DISTROTYPE = centos ] && [ $SOCFAMILY = LS ]; then
 	echo Installing restool in $DISTROTYPE ...
-- 
2.27.0

